.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!--
+If your issue is a usage question, submit it to the user mailing-list
+users@riot-os.org or the developer mailing-list devel@riot-os.org.
+-->
+
+#### Description
+<!-- Example: Cannot build gnrc_networking application for samr21-xpro
+board. -->
+
+#### Steps to reproduce the issue
+<!--
+Try to describe as precisely as possible here the steps required to reproduce
+the issue. Here you can also describe your hardware configuration, the network
+setup, etc.
+-->
+
+#### Expected results
+<!-- Example: The gnrc_networking application builds on samr21-xpro. -->
```

Comment:
I stumble now the second time over this sentence and for the second time I only realized after the second read-through, that this is an example. Don't know why though with this particular line. The other examples are not that much a problem.